hasura/graphql-engine
# update allowed queries in schema cache on add/drop queries to collection (fix #2221)

### Description



### Affected components 


- Server
- Tests

### Related Issues



fix #2221 

### Solution and Design


If a collection modified is part of allowlist then refresh allowlist in schema cache.

### Steps to test and verify


Create a allowlist query in console. Update the query. Check if it is reflected in queries.
